cookiecutter/vcs.py

```
@@ -115,9 +115,19 @@ def clone(repo_url, checkout=None, clone_to_dir=".", no_input=False):
         prompt_and_delete_repo(repo_dir, no_input=no_input)
 
     if repo_type in ['git', 'hg']:
-        subprocess.check_call([repo_type, 'clone', repo_url], cwd=clone_to_dir)
-        if checkout is not None:
-            subprocess.check_call([repo_type, 'checkout', checkout],
-                                  cwd=repo_dir)
+        try:
+            subprocess.check_output(
+                [repo_type, 'clone', repo_url],
+                cwd=clone_to_dir,
+                stderr=subprocess.STDOUT,
+            )
+            if checkout is not None:
```

You make a good point, I think the code _would_ be less surprising, but at the expense of swallowing any unexpected `subprocess` failures.
😕